t0k2/src/building/Address.java

```diff
@@ -0,0 +1,39 @@
+package building;
+
+
+public class Address {
+	private String street;
+	private int addressNumber;
+	
+	@Override
+	public String toString() {
+		return "Address [street=" + street + ", addressNumber=" + addressNumber + "]";
+	}
+
+	private Address(Builder b){
```

Comment:
Here you are coupling the Address to the builder. The Address shouldn't know anything about the builder but the other way around